Source/WebCore/html/HTMLVideoElement.cpp

```diff
@@ -151,12 +151,8 @@ void HTMLVideoElement::attributeChanged(const QualifiedName& name, const AtomStr
         HTMLMediaElement::attributeChanged(name, oldValue, newValue, attributeModificationReason);
 
 #if PLATFORM(IOS_FAMILY) && ENABLE(WIRELESS_PLAYBACK_TARGET)
-        if (name == webkitairplayAttr) {
-            bool disabled = false;
-            if (equalLettersIgnoringASCIICase(attributeWithoutSynchronization(HTMLNames::webkitairplayAttr), "deny"_s))
-                disabled = true;
-            mediaSession().setWirelessVideoPlaybackDisabled(disabled);
-        }
+        if (name == webkitairplayAttr)
+            mediaSession().setWirelessVideoPlaybackDisabled(hasWirelessPlaybackTargetDisabled());
```

Comment:
`isWirelessPlaybackTargetDisabled()` isn't a boolean checks
checks `equalLettersIgnoringASCIICase(attributeWithoutSynchronization(HTMLNames::webkitairplayAttr), "deny"_s)`

this change only moved the logic so it's all done in the same place.

However, it made little sense to enable AirPlay there, when the Remote Playback API would have disabled it. So I've combined all those together so that it's no consistent